scripts/rmc.py

```diff
@@ -51,9 +51,10 @@ def ensure_dependencies_in_path():
         ensure(is_exe(program), f"Could not find {program} in PATH")
 
 # Assert a condition holds, or produce a user error message.
-def ensure(condition, message, retcode=1):
+def ensure(condition, message=None, retcode=1):
```

Comment:
when do we not want to have a message

Reply:
In `cargo-rmc`, we manually print out possibly multiple error messages, so we don't need the `ensure` itself to print a message.

Reply:
https://github.com/model-checking/rmc/pull/355/files#diff-876efdc1d5780b01eb535f6e409bbfe28bea1b5876d28456f3f7b2e03b7e6decR211